docs/sources/reference/config-blocks/import.file.md

```diff
@@ -56,11 +74,71 @@ declare "add" {
 
 {{< /collapse >}}
 
-{{< collapse title="importer.alloy" >}}
+
+This example imports a module from a file inside of a module that is imported via [import.git][]:
+
+{{< collapse title="main.alloy" >}}
+
+```alloy
+import.git "math" {
+  repository = "https://github.com/wildum/module.git"
+  path       = "relative_math.alloy"
+}
+
+math.add "default" {
+  a = 15
+  b = 45
+}
+```
+
+{{< /collapse >}}
+
+{{< collapse title="relative_math.alloy" >}}
+
+```alloy
+import.file "lib" {
+  filename = file.path_join(module_path, "lib.alloy")
```

Comment:
Do we need to ensure users cannot escape outside the sandbox and start accessing files outside of the git repo checkout in this example? What's been our threat model / security approach to modules?

Reply:
they can already do it. If an attacker accesses the imported module, he can modify it for malicious purposes. There were some discussions around module protection levels months ago but it was not prioritized

Reply:
Yeah, I guess that's our approach - keep your config files safe from modification. 
I'd like to doc that: https://github.com/grafana/alloy/pull/1746/files